src/main/java/org/broadinstitute/hellbender/engine/ReferenceFileSource.java

```diff
@@ -3,10 +3,8 @@
 import htsjdk.samtools.SAMSequenceDictionary;
 import htsjdk.samtools.reference.ReferenceSequence;
 import org.broadinstitute.hellbender.exceptions.GATKException;
-import org.broadinstitute.hellbender.utils.SimpleInterval;
```

Comment:
import org.broadinstitute.hellbender.utils.iterators.ByteArrayIterator; does not seem to be present either.  I just see:

package org.broadinstitute.hellbender.engine;

import htsjdk.samtools.SAMSequenceDictionary;
import htsjdk.samtools.reference.ReferenceSequence;
import org.broadinstitute.hellbender.exceptions.GATKException;
import org.broadinstitute.hellbender.utils.Utils;
import org.broadinstitute.hellbender.utils.fasta.CachingIndexedFastaSequenceFile;

import java.io.File;
import java.io.IOException;
import java.util.Iterator;

Please let me know how I can resolve these discrepancies.